numpy/lib/function_base.py

```diff
@@ -1094,6 +1094,18 @@ def diff(a, n=1, axis=-1):
     axis : int, optional
         The axis along which the difference is taken, default is the
         last axis.
+    prepend : array_like
+        Values to prepend to the beginning of "a" along axis before
+        performing the difference.  Scalar values are expanded to
+        arrays with length 1 in the direction of axis and the shape
+        of the input array in along all other axes.  Otherwise the
+        dimension and shape must match "a" except along axis.
+    append : array_like
```

Comment:
I'd be in favor of combining these into `prepend, append : array_like`, since the description is almost identical